app/views/sequences/batch_load/genbank/_batch_load.html.erb

```diff
@@ -0,0 +1,36 @@
+<div>
+  <h2> Genbank batch load </h2>
+
+  <div class="flexbox full_viewport_whitespace">
+    <div class="item item1 one_third_width">
+      <h3> Description and requirements </h3>
+
+      <p>Creates sequences and origin relationships and attaches at most 2 identifiers to each sequence</p>
+      <ul>
+        <li>A tab-delimited, UTF-8 compatible text file is required.</li>
+        <li>Column headers are "filename" and "file_content" (see <%= link_to('sample template', asset_path('batch_load_templates/sequences/sequences_genbank_batch_load.tab'), target: '_blank') -%>).</li>
+        <li>Multiple uploads of the same data will fail.</li>
+        <li>Assumes a namespace with name "DRMSequenceId" and shortname "DRMSEQID" exists</li>
```

Comment:
Don't worry about it now, but ultimately it would be nice to make this a user-definable parameter submitted on load.